res/examples/strong_equivalence/squares/README.md

```diff
@@ -0,0 +1,10 @@
+# `squares`
+
+## Usage
+```
+mkdir out
+anthem verify --equivalence strong squares.1.lp squares.2.lp
+```
+
+## Origin
+This is an interesting problem - increasing the interval for X increases the difficulty greatly.
```

Comment:
I don't really think that qualifies as an origin. If we include this problem into our paper, I'd reference that paper. Otherwise, I'd not report an origin.